vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/permission_rules/per_hour_limit_frequency_cap.cc

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/permission_rules/per_hour_limit_frequency_cap.h"
+#include "bat/ads/internal/frequency_capping/frequency_capping.h"
+#include "bat/ads/ads_client.h"
+#include "bat/ads/internal/time.h"
+#include "bat/ads/internal/client.h"
+#include "bat/ads/internal/ads_impl.h"
+
+#include "bat/ads/ad_info.h"
+
+namespace ads {
+
+bool PerHourLimitFrequencyCap::IsAllowed() {
```

Comment:
As above should be `AdsPerHourFrequencyCap` to be consistent with naming, please rename class and tests too

Reply:
Done